skupper-router.spec.rpkg

```diff
@@ -0,0 +1,144 @@
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# This .spec file uses `rpkg` (https://pagure.io/rpkg-util) to provide
+# pleasant user experience to developers.
+#
+#  Command                  Description
+# `rpkg srpm`              Creates a *.src.rpm file in /tmp/rpkg (exact path is printed)
+# `rpkg local --nocheck`   Builds a *.rpm for your system in /tmp/rpkg (exact path is printed)
+#
+# See `man rpkg` for more commands.
+
+%global proton_minimum_version 0.34.0
+%global libwebsockets_minimum_version 3.0.1
+%global libnghttp2_minimum_version 1.33.0
+
+Name:          skupper-router
+Version:       {{{ git_dir_version }}}
+Release:       2.0.0%{?dist}
+Summary:       Router for Skupper.io
+License:       ASL 2.0
+URL:           https://skupper.io
+VCS:           {{{ git_dir_vcs }}}
+
+BuildRequires: cmake
+BuildRequires: python3-devel
+BuildRequires: openssl-devel
+BuildRequires: qpid-proton-c-devel >= %{proton_minimum_version}
+BuildRequires: libwebsockets-devel >= %{libwebsockets_minimum_version}
+BuildRequires: libnghttp2-devel >= %{libnghttp2_minimum_version}
+# man pages --help
+BuildRequires: asciidoc
+BuildRequires: python3-qpid-proton >= %{proton_minimum_version}
+
+Source: {{{ git_dir_pack }}}
+
+%description
+A lightweight message router, written in C and built on Qpid Proton, that provides flexible and scalable interconnect backend for Skupper.io Level 7 Virtual Application Network.
+
+%prep
+{{{ git_dir_setup_macro }}}
+
+%build
+%cmake -DUSE_LIBWEBSOCKETS=ON -DUSE_LIBNGHTTP2=ON -DPython_EXECUTABLE=%{python3}
+%cmake_build --target all --target man
+
+%install
+%cmake_install
+
+%check
+%ctest
+
+# TODO: name?
+%package server
+Summary:  The skrouterd deamon
+Requires: python3
+Requires: qpid-proton-c >= %{proton_minimum_version}
+Requires: libwebsockets >= %{libwebsockets_minimum_version}
+Requires: libnghttp2 >= %{libnghttp2_minimum_version}
+
+%description server
+%{summary}.
+
+%files server
+/usr/sbin/skrouterd
+
+%config /etc/skupper-router/skrouterd.conf
+%config /etc/sasl2/skrouterd.conf
+
+%{python3_sitelib}/skupper_router/
+%{python3_sitelib}/skupper_router_site.py
+%{python3_sitelib}/__pycache__/skupper_router_site.*.pyc
+%{python3_sitelib}/skupper_router-*.egg-info
+
+# TODO: skupper-router-tools depend on tools/command.py, etc. in skupper_router_internal/
+/usr/lib/skupper-router/python/skupper_router_internal/
+
+/usr/share/man/man5/skrouterd.conf.5.gz
+/usr/share/man/man8/skrouterd.8.gz
+
+/usr/share/skupper-router/html/index.html
+
+# TODO: package -tools in fact depends on -server, see TODO above
+# TODO: what to do with scrapper tool?
```

Comment:
scrapper?

Reply:
Darn - is it too late to rename the scraper tool, 'cause "scrapper" is so much better ;)

Reply:
In any case, scraper should not be packaged, because it never before was packaged.